receiver/kymastatsreceiver/documentation.md

```diff
@@ -22,11 +22,11 @@ The resource status conditions. Possible metric values for condition status are
 
 #### Attributes
 
-| Name | Description | Values |
-| ---- | ----------- | ------ |
-| reason | The reason of the resource condition status. | Any Str |
-| status | The status of the resource condition. | Any Str |
-| type | The type of the resource condition. | Any Str |
+| Name | Description | Values | Optional |
```

Comment:
Some tables have a column named "Enabled" and some have "Optional". Are these columns different from each other?